.circleci/config.yml

```diff
@@ -41,61 +44,50 @@ jobs:
       # one for when we can get CircleCI Clojars deploys working again
       # - run: ./.circleci/bin/deploy_clojars.sh
 
-  long-build:
+  deploy:
     docker:
       # CircleCI maintains a library of pre-built images
       # documented at https://circleci.com/docs/2.0/circleci-images/
       - image: circleci/clojure:lein-2.9.1
 
-
     working_directory: ~/crux
-
-    environment:
-      LEIN_ROOT: "true"
-      # Customize the JVM maximum heap limit
-      JVM_OPTS: -Xmx3200m
-      CRUX_TS_DEVICES: "true"
-      CRUX_TS_WEATHER: "true"
-
     steps:
-      - checkout
-
-      # Download and cache dependencies
-      - restore_cache:
-          keys:
-          - v1-dependencies-{{ checksum "project.clj" }}
-          # fallback to using the latest cache if no exact match is found
-          - v1-dependencies-
-      - run: lein sub install
-      - run: lein deps
-      - run:
-          name: Install datasets for ts-weather-test and ts-devices-test
-          command: |
-            mkdir -p crux-test/resources/ts/data
-            cd crux-test/resource/ts/data/
-            wget https://timescaledata.blob.core.windows.net/datasets/weather_small.tar.gz
-            tar -xzf weather_small.tar.gz
-            wget https://timescaledata.blob.core.windows.net/datasets/devices_small.tar.gz
-            tar -xzf devices_small.tar.gz
-      - save_cache:
-          paths:
-            - ~/.m2
-          key: v1-dependencies-{{ checksum "project.clj" }}
-
-      - run: lein do version, sub test, check
+        - checkout
+        - restore_cache:
+            keys:
+              - v1-dependencies-{{ checksum "project.clj" }}
+              - v1-dependencies-
+        - run: lein sub install
+        - run: lein deps
+
+        - save_cache:
+            paths:
+              - ~/.m2
+            key: v1-dependencies-{{ checksum "project.clj" }}
+
+        - run: sh ./crux-bench/bin/download_dataset.sh
+        - run: lein sub -s crux-bench uberjar
+        - run:
+            command: |
+              echo 'export BUILD_ARG="--build-arg crux_commit=$CIRCLE_SHA1"' >> $BASH_ENV
+              echo 'export BUILD_TAGS="master,commit-$CIRCLE_SHA1"' >> $BASH_ENV
+
+        - setup_remote_docker
```

Comment:
guessing this activates configuration elsewhere in CircleCI?

Reply:
Yeah, part of which is to set it up for deploying via docker